WORKSPACE

```diff
@@ -26,12 +26,14 @@ http_archive(
 # b) get the sha256 hash of the commit by running:
 #    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
 #    and update the sha256 with the result.
+# FIXME: revert before https://github.com/tensorflow/tensorflow/commit/62544da4337a92afba8f993b8beca11592b45f16
+#  to avoid newly added //tensorflow/core/tpu:tpu_executor_dlsym_initializer building problem on windows
```

Comment:
I might fix myself...

Reply:
Note we constantly update the TF github hash. e.g., I just sent out https://github.com/google/jax/pull/4845 which updates it to TF head.

You'll need to make your code work with TF head.